catalog.html

```diff
@@ -1,236 +1,241 @@
-        <form class="subscription-form" action="https://echo.htmlacademy.ru/" method="post">
-          <label for="subscription-email">Email</label>
-          <input class="field" placeholder="email@example.ru" type="email" name="subscription-email"
-            id="subscription-email" required>
-          <button class="button subscription-button" type="submit">Подписаться</button>
-        </form>
-      </section>
-    </main>
-    <footer class="main-footer">
-      <ul class="social-list">
-        <li class="social-item">
-          <a href="https://vk.com/htmlacademy">
-            <span class="visually-hidden">Vkontakte</span>
+<body>
+  <header class="mane-header">
+    <a class="logo" href="index.html">
+      <img src="images/logo-sedona.svg" width="140" height="70" alt="логотип города Sedona">
+    </a>
+    <nav class="navigation">
+      <ul class="site-navigation">
+        <li class="site-navigation-item">
+          <a class="navigation-link" href="index.html">Главная</a>
+        </li>
+        <li class="site-navigation-item">
+          <a class="navigation-link" href="about.html">О седоне</a>
+        </li>
+        <li class="site-navigation-item">
+          <a class="navigation-link" href="catalog.html">Гостиницы</a>
+        </li>
+      </ul>
+      <ul class="menu">
+        <li class="menu-item">
+          <a class="menu-link" href="#">
+            <span class="visually-hidden">Поиск гостиницы</span>
           </a>
         </li>
-        <li class="social-item">
-          <a href="https://t.me/htmlacademy">
-            <span class="visually-hidden">Telegram</span>
+        <li class="menu-item">
+          <a class="menu-link" href="#">
+            <span class="visually-hidden">избранное</span>
+            <span class="notification-badge">12</span>
           </a>
         </li>
-        <li class="social-item">
-          <a href="https://www.youtube.com/user/htmlacademyru">
-            <span class="visually-hidden">Youtube</span>
+        <li class="menu-item">
+          <a class="menu-button" href="#">Хочу сюда!</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="main-inner">
+    <section class="filter-background">
+      <h1 class="inner-header-title">Гостиницы Седоны</h1>
+      <ul class="breadcrumbs-item">
+        <li class="breadcrumbs-item">
+          <a class="breadcrumbs-link" href="index.html">
+            <span class="visually-hidden">Главная</span>
           </a>
         </li>
+        <li class="breadcrumbs-item">
+          <a class="breadcrumbs-link">Гостиницы</a>
+        </li>
       </ul>
-      <a class="contacts-phone" href="tel:+78128121212">+7(812)812-12-12</a>
-      <a class="logo-academy" href="https://htmlacademy.ru/">
-        <img src="images/htmlacademy.svg" width="115" height="33" alt="Логотип HTML-academy">
-      </a>
-    </footer>
-  </body>
+      <h3 class="visually-hidden">Список гостиниц с фильтрами</h3>
```

Comment:
```suggestion
      <h2 class="visually-hidden">Список гостиниц с фильтрами</h2>
```
Иначе нарушается иерархия заголовков.
![image](https://github.com/htmlacademy-htmlcss/2518753-sedona-1/assets/17790771/848e2c29-32a3-43e7-804e-47a1afef636c)